.github/workflows/tests.yml

```diff
@@ -39,3 +39,67 @@ jobs:
         npm rebuild node-sass
     - name: Run tests
       run: yarn test
+
+  visual_tests:
+    name: Frontend Visual Tests
+    needs: test
+    if: ${{ github.event_name == 'pull_request' && needs.test.result == 'success' }}
+    runs-on: ubuntu-latest
+    outputs:
+      percy_url: ${{ steps.extract-url.outputs.percy_url }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+      - name: Cache Node.js modules
+        uses: actions/cache@v4
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+      - name: Install dependencies
+        run: |
+          yarn --frozen-lockfile
+          npm rebuild node-sass
+      - name: Extract jsdocs and environment info 
+        run: |
```

Comment:
I think we dont need these pipes in these two steps right? (extract jsdocs and run visual tests). As we only have a one-line command

Reply:
Oh, Sorry for the oversight. I've updated it accordingly :+1: